entrypoint.sh

```diff
@@ -18,7 +18,11 @@ if ! grep ${SSH_USER} /etc/passwd ; then
 
 	# Copy the authorized_keys
 	touch /home/${SSH_USER}/.ssh/authorized_keys
-	echo "$SSH_KEY" >> /home/${SSH_USER}/.ssh/authorized_keys
+	IFS=',' read -a keys <<< "${SSH_KEY}"
+    	for key in "${keys[@]}"
```

Comment:
@ruiztulio 
Not actually in the line [entrypoint.sh#4](https://github.com/Vauxoo/docker-ssh/pull/3/files/f9200617ef616f3fb02791c6467786b0ad861c35#diff-b958f585a04af5ee2087610ea7180f34R4) has a if.

The if sentences check if the user exists

`if ! grep ${SSH_USER} /etc/passwd ; then`

For this reason the script only ran one time